sentry/README.md

```diff
@@ -7,18 +7,20 @@
 # Sentry Rust SDK: sentry
 
 This crate provides support for logging events and errors / panics to the
-[Sentry](https://sentry.io/) error logging service.  It integrates with the standard panic
+[Sentry] error logging service. It integrates with the standard panic
 system in Rust as well as a few popular error handling setups.
 
+[Sentry]: https://sentry.io/
+
 ## Quickstart
 
-The most convenient way to use this library is the [`sentry::init`] function,
+The most convenient way to use this library is via the [`sentry::init`] function,
 which starts a sentry client with a default set of integrations, and binds
 it to the current [`Hub`].
 
 The [`sentry::init`] function returns a guard that when dropped will flush Events that were not
-yet sent to the sentry service.  It has a two second deadline for this so shutdown of
```

Comment:
😱 😂

Reply:
(because seeing this comment anywhere else then in a side-by-side diff makes little sense: yes this is a comment on single-vs-double space sentence ending 😄 )

Reply:
😢 I'm glad you pointed this out actually, this is actually a bit of a dilemma for me: Some places in the docs use double spaces, which some auto-formatter that I need to hunt down has been aggressively converting to single spaces. Should I be using 2 spaces, or 1 space?

Reply:
honestly, i think it doesn't matter and personally I'm fine with mixed.  and if we want consistency we should have an auto-formatter running in CI.  I just found it funny that you changed them, but if it was auto-formatted that explains :)